tests/components/flux_led/__init__.py

```diff
@@ -76,6 +85,19 @@
     remote_access_port=8816,
 )
 
+DEFAULT_ENTRY_DATA = {
```

Comment:
Discovery would cause a reload at startup previous so the tests now set the data to avoid the double reload